ocaml_version.ml

```diff
@@ -304,16 +311,8 @@ module Configure_options = struct
       | _, Error b -> Error b
       | Error a, _-> Error a) (Ok [])
 
-  let to_configure_flag {major; minor; _} o =
-    if major < 5 && minor < 8 then (* pre autoconf *)
-      match o with
-      | `Afl -> "-afl-instrument"
-      | `Flambda -> "-flambda"
-      | `Default_unsafe_string -> "-default-unsafe-string"
-      | `Force_safe_string -> "-force-safe-string"
-      | `Frame_pointer -> "-with-frame-pointer"
-      | _ -> ""
-    else (* post autoconf *)
+  let to_configure_flag t o =
+    if Has.autoconf t then
```

Comment:
shouldn't it be `not (Has.autoconf t)` instead?

Reply:
The branches are flipped, compared to `master`. So the "has autoconf" / "post autoconf" branch is the `then` one now.

Reply:
oh, I see, code review on my phone was a bad idea.. I'm so sorry for the disturbance.

Reply:
no problem I was confused with these comparisons as well (which prompted me to write a couple tests)